common/networkProfile.go

```diff
@@ -0,0 +1,30 @@
+package common
+
+// NetworkProfile is used in ThrottleNetwork.
+type NetworkProfile struct {
+	// Minimum latency from request sent to response headers received (ms).
+	Latency float64 `js:"latency"`
+
+	// Maximal aggregated download throughput (bytes/sec). -1 disables download throttling.
+	DownloadThroughput float64 `js:"downloadThroughput"`
+
+	// Maximal aggregated upload throughput (bytes/sec). -1 disables upload throttling.
+	UploadThroughput float64 `js:"uploadThroughput"`
+}
+
+// GetNetworkProfiles returns NetworkProfiles which are ready to be used to
+// throttle the network with page.throttleNetwork.
+func GetNetworkProfiles() map[string]NetworkProfile {
+	return map[string]NetworkProfile{
+		"Slow 3G": {
+			DownloadThroughput: ((500 * 1000) / 8) * 0.8,
```

Comment:
Should we add a comment to clarify what these magic numbers mean?

Reply:
Tbh, i'm not sure what these numbers mean. They were copied from [here](https://github.com/microsoft/playwright/issues/6038#issuecomment-812521882) and [here](https://github.com/puppeteer/puppeteer/blob/0a41f8d01e85ff732fdd2e50468bc746d7bc6475/packages/puppeteer-core/src/cdp/PredefinedNetworkConditions.ts#L41-L52). What about:

```js
		"Slow 3G": {
			Download: 50000,
			Upload:   50000,
			Latency:  2000,
		},
		"Fast 3G": {
			Download: 180000
			Upload:   84375,
			Latency:  562.5,
		},
```

Reply:
LGTM. Here is more info on what the numbers mean: https://stackoverflow.com/questions/48367042/in-chrome-dev-tools-what-is-the-speed-of-each-preset-option-for-network-throttl.

Reply:
Thanks for the link. I quite the explanation, so i'll copy it over into here instead of converting them to the final value as I suggested in the comment [above](https://github.com/grafana/xk6-browser/pull/1098#discussion_r1391225957). Having them broken down into smaller steps might help us to create new predefined profiles later on.

Reply:
Added the notes on what the numbers mean in 9118e7764a6895c5fec126720e071ccf171e5cd1